docs/generated/packages/jest/executors/jest.json

```diff
@@ -105,7 +108,8 @@
       },
       "passWithNoTests": {
         "description": "Will not fail if no tests are found (for example while using `--testPathPattern`.) (https://jestjs.io/docs/cli#--passwithnotests)",
-        "type": "boolean"
+        "type": "boolean",
+        "x-priority": "important"
```

Comment:
Is this really something folks will want to set almost all of the time they use this executor? 
To me it seems like `passWithNoTests` gets enabled as soon as there happens to be no tests which isn't that often, right? Most generators scaffold sample tests